src/ussd_popi_rapidpro.js

```diff
@@ -1,27 +1,173 @@
 go.app = function() {
+    var _ = require("lodash");
+    var moment = require("moment");
     var vumigo = require("vumigo_v02");
+    var utils = require("seed-jsbox-utils").utils;
     var App = vumigo.App;
     var Choice = vumigo.states.Choice;
+    var EndState = vumigo.states.EndState;
+    var JsonApi = vumigo.http.api.JsonApi;
     var MenuState = vumigo.states.MenuState;
+    var PaginatedState = vumigo.states.PaginatedState;
 
     var GoNDOH = App.extend(function(self) {
         App.call(self, "state_start");
         var $ = self.$;
 
         self.init = function() {
+            self.rapidpro = new go.RapidPro(
+                new JsonApi(self.im, {headers: {'User-Agent': ["Jsbox/NDoH-Public"]}}),
```

Comment:
does it use the public line, or have its own line?

Reply:
Nope, copy and paste error, good catch :+1: